tests/ProjectTestCase.php

```diff
@@ -8,7 +8,7 @@ class ProjectTestCase extends TestCase
 {
     protected function setUpDir($projectDir)
     {
-        $this->rootDir = realpath($projectDir . '/prj');
+        $this->rootDir = realpath($projectDir . '/Asset');
```

Comment:
That's not directly related, but what do you think about this change?

Reply:
asset is something you will use in your project, like style or image.
here, it's a fixture project.